### PR TITLE
US103199 - add sorting to quick eval

### PR DESCRIPTION
**Pull request by AlexBedley**
I notice it overwrote a bunch of the specific semver required versions with commit hashes versions... that doesn't feel right. What am I missing??

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -2379,58 +2379,59 @@
...
-        "d2l-icons": "github:BrightspaceUI/icons#semver:^6",
-        "d2l-polymer-behaviors": "github:Brightspace/d2l-polymer-behaviors-ui#semver:^2"
+        "d2l-colors": "github:BrightspaceUI/colors#6a1c414d6f3685e3c286aff4147ede037bdbb693",
+        "d2l-icons": "github:BrightspaceUI/icons#7f91e70700322582374753d744442c58dd9dd071",
+        "d2l-polymer-behaviors": "github:Brightspace/d2l-polymer-behaviors-ui#ced40c46cbaa2710862c82e03ff373660b8223d9"
       }
     },
     "d2l-activities": {
-      "version": "github:BrightspaceHypermediaComponents/activities#6bb2b3392d506dd0b52389587cb97b02c1048e50",
+      "version": "github:BrightspaceHypermediaComponents/activities#c5d1defd3bd1fb814247fc7fa7c75f45c37bf368",
```
Comment (AlexBedley) [inline package-lock.json]: this commit hash is correct at least https://github.com/BrightspaceHypermediaComponents/activities/commit/c5d1defd3bd1fb814247fc7fa7c75f45c37bf368

**Comment by svanherk** [comment]
We see this if you're not using the latest `npm`.  I have `npm` `6.8.0`

**Comment by AlexBedley** [comment]
> We see this if you're not using the latest `npm`. I have `npm` `6.8.0`

```
abedley ~/dev/d2l/brightspace-integration (US103199)
λ npm --version
6.4.1
```

I'll try updating

**Comment by AlexBedley** [comment]
Closing because my changes got captured in https://github.com/Brightspace/brightspace-integration/pull/1257

**Pull request closed without merging**